### PR TITLE
Convert xml to json

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
Fieldwork were still using legacy XML messages to receive instructions from RM. We wanted to modernise and use JSON.

# What has changed
Migrated from XML to JSON

# How to test?
Run the acceptance tests.

# Links
Trello: https://trello.com/c/VsrzpkgG/492-xml-to-json-for-field-messages-create-and-close-13

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[XML to JSON for field messages - Create and Close (13)](https://trello.com/c/VsrzpkgG/492-xml-to-json-for-field-messages-create-and-close-13)

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/InvalidAddressReceiverIT.java [hunk 1]
```diff
@@ -70,7 +68,7 @@ public void setUp() {
 
   @Test
   public void testInvalidAddressMessageFromNonFieldChannelEmitsMessageToField()
-      throws InterruptedException, JAXBException, JsonProcessingException {
+      throws InterruptedException, JAXBException, IOException {
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/fwmtadapter/messaging/InvalidAddressReceiverIT.java]: `JAXBException` not needed anymore?

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiverIT.java [hunk 2]
```diff
@@ -60,46 +59,36 @@ public void testReceiveMessage() throws InterruptedException, JAXBException {
...
-        .isEqualTo(fieldworkFollowup.getTreatmentCode());
-    assertThat(actionInstruction.getActionRequest().getCeExpectedResponses())
+            "fieldOfficerId");
+    assertThat(actionInstruction.getCeExpectedCapacity())
         .isEqualTo(fieldworkFollowup.getCeExpectedCapacity());
   }
 
   @Test
-  public void testReceiveMessageCE() throws InterruptedException, JAXBException {
+  public void testReceiveMessageCE() throws InterruptedException, JAXBException, IOException {
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiverIT.java]: `JAXBException ` not needed anymore?

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/UacUpdatedReceiverIT.java [hunk 3]
```diff
@@ -65,13 +63,14 @@ public void setUp() {
   }
 
   @Test
-  public void testGoodReceiptMessage()
-      throws InterruptedException, JAXBException, JsonProcessingException {
+  public void testGoodReceiptMessage() throws InterruptedException, JAXBException, IOException {
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/fwmtadapter/messaging/UacUpdatedReceiverIT.java]: `JAXBException` not needed anymore

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiverIT.java (hunk 2 above)
Comment (NickGrantONS) [reply src/test/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiverIT.java]: fixed

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/InvalidAddressReceiverIT.java (hunk 1 above)
Comment (NickGrantONS) [reply src/test/java/uk/gov/ons/census/fwmtadapter/messaging/InvalidAddressReceiverIT.java]: fixed

File: src/test/java/uk/gov/ons/census/fwmtadapter/messaging/UacUpdatedReceiverIT.java (hunk 3 above)
Comment (NickGrantONS) [reply src/test/java/uk/gov/ons/census/fwmtadapter/messaging/UacUpdatedReceiverIT.java]: gone

**Pull request merged**